### ISSUE TITLE
Hippocampal malrotation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7770]
**Preferred term label:**
Hippocampal malrotation

**Synonyms**

**Definition (free text, please give PubMed ID)**
A neuroimaging feature characterized by rounded hippocampal shape, deep collateral, or occipitotemporal sulcus, and medial localization of the hippocampus.
PMID: [33916495](https://pubmed.ncbi.nlm.nih.gov/33916495)
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 617929 
PMID: 29394991  3/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
